tests/bot/exts/info/test_information.py

```diff
@@ -281,9 +281,13 @@ async def test_create_user_embed_uses_string_representation_of_user_in_title_if_
         """The embed should use the string representation of the user if they don't have a nick."""
         ctx = helpers.MockContext(channel=helpers.MockTextChannel(id=1))
         user = helpers.MockMember()
+        public_flags = unittest.mock.MagicMock()
+        public_flags.__iter__.return_value = iter(())
```

Comment:
What's the reason for patching the iter?

Reply:
Seems like the default behavior was enough, it has been removed in 9a0b7643.